api/instance/v1/instance_utils.go

```diff
@@ -57,3 +57,27 @@ func (s *API) DetachIP(req *DetachIPRequest, opts ...scw.RequestOption) (*Detach
 
 	return &DetachIPResponse{IP: ipResponse.IP}, nil
 }
+
+// UpdateIPRequest contains the parameters to updaten an IP
+type UpdateIPRequest struct {
+	Zone    utils.Zone `json:"-"`
+	IPID    string     `json:"-"`
+	Reverse *string    `json:"reverse"`
```

Comment:
Add comment that specify:
` If Reverse is an empty string reverse will be remove`